scripts/ubuntu/networking.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh -eux
+
+echo "Disabling automatic udev rules for network interfaces in Ubuntu"
+# Disable automatic udev rules for network interfaces in Ubuntu,
+# appends cmdline options to the linux kernel
+sed -i 's/append\=\"/append\=\"net\.ifnames\=0\ /' /etc/yaboot.conf\
+  || echo "Yaboot not found!"
+sed -i 's/GRUB_CMDLINE_LINUX_DEFAULT\=\"/GRUB_CMDLINE_LINUX_DEFAULT\=\"net\.ifnames\=0 /' /etc/default/grub\
+  && grub-mkconfig -o /boot/grub/grub.cfg\
+  || echo "Grub2 not found!"
+
+# Fix ethernet address for network interfaces
+sed -i 's/auto\ en.*/auto\ eth0/' /etc/network/interfaces
+sed -i 's/iface\ en.*/iface\ eth0\ inet\ dhcp/' /etc/network/interfaces
+
+# Adding a 5 sec delay to the interface up, to make the dhclient happy
+echo "pre-up sleep 5" >> /etc/network/interfaces;
+
+# Disable cloud-init networking configuration
+mkdir -p /etc/cloud/cloud.cfg.d
+echo "network: {config: disabled}" > /etc/cloud/cloud.cfg.d/99-disable-network-config.cfg
```

Comment:
Keeping cloud-init configuring networks was causing problems for me, the instances never got their networking set up correctly. Right now this PR sets an `eth0` interface to dhcp manually, but I can see now that this will cause problems when we get multiple interfaces. I'll work on getting this working without having to set this.